tests/e2e/medic-collect.js

```diff
@@ -91,43 +90,42 @@ Connection: close\r
   });
 
   describe('with User-Agent header', () => {
-    it('is prompted for auth details if not supplied', () => {
+    it('is prompted for auth details if not supplied', async () => {
       // when
-      return rawHttpRequest(
+      const res = await rawHttpRequest(
         `HEAD /${dbName}/_design/medic/_rewrite/add?deviceID=imei%3A357578064823168 HTTP/1.1\r
 X-OpenRosa-Version: 1.0\r
 Date: ${new Date().toISOString()}\r
 Host: ${host}:${port}\r
 User-Agent: Dalvik/1.6.0 (Linux; U; Android 4.4.2; TECNO-Y4 Build/KOT49H) org.medicmobile.collect.android/SNAPSHOT\r
 Connection: close\r
 \r\n`
-      ).then(res => {
+      );
         // then
-        assert.equal(res.statusCode, 401, JSON.stringify(res));
-        assert.equal(
-          res.headers['WWW-Authenticate'],
-          'Basic realm="Medic Mobile Web Services"',
-          JSON.stringify(res)
-        );
-      });
+      assert.equal(res.statusCode, 401, JSON.stringify(res));
+      assert.equal(
+        res.headers['WWW-Authenticate'],
+        'Basic realm="Medic Mobile Web Services"',
+        JSON.stringify(res)
+      );
     });
 
-    it('can fetch a list of forms', () => {
+    it('can fetch a list of forms', async () => {
       // when
-      return rawHttpRequest(
+      const res = await rawHttpRequest(
         `GET /api/v1/forms HTTP/1.1\r
 X-OpenRosa-Version: 1.0\r
 Date: ${new Date().toISOString()}\r
 Host: ${host}:${port}\r
 User-Agent: Dalvik/1.6.0 (Linux; U; Android 4.4.2; TECNO-Y4 Build/KOT49H) org.medicmobile.collect.android/SNAPSHOT\r
 Connection: close\r
 \r\n`
-      ).then(res => {
+      );
         // then
```

Comment:
Please remove this comment as it was more appropriate when there was a `then` function call here.